sections/dir.zsh

```diff
@@ -21,14 +22,36 @@ SPACESHIP_DIR_COLOR="${SPACESHIP_DIR_COLOR="cyan"}"
 spaceship_dir() {
   [[ $SPACESHIP_DIR_SHOW == false ]] && return
 
-  local dir
+  local dir trunc_prefix
 
   # Threat repo root as a top-level directory or not
   if [[ $SPACESHIP_DIR_TRUNC_REPO == true ]] && spaceship::is_git; then
     local git_root=${$(git rev-parse --absolute-git-dir):h}
```

Comment:
This change has been reverted on the current master. Because this requires at least `git v2.13.0`. Because `--absolute-git-dir` [was added with that release.](https://github.com/git/git/commit/a2f5a8762693d5af6dbbca714d882a21d7ba0b75)